src/handlebar-helpers.js

```diff
@@ -83,6 +83,14 @@ const helpers = {
   jsonDoubleStringify: function (data, space) {
     // if you want to use this, you must use our strTemplate, and you must not put quotes around your template (ie. `my-field: {{ jsonStringify my-json }}` is valid but `my-field: "{{ jsonStringify my-json }}"` is not)
     return JSON.stringify(JSON.stringify(data, null, space));
+  },
+  pick: function (...args) {
+    return args.find((value) => {
+      // null or undefined
+      if (value == null) return false;
+      if (value === '') return false;
```

Comment:
my preference would be to be explicit and use `===`

```suggestion
      // if value is nullish or empty string, return false
      if (value ===  null || value ===  undefined || value ===  '') return false;
      // else value is not nullish and should return true
```

Reply:
updated